### DISCUSSION TITLE
Provide shortcut link to examples on the doc landing page

### DISCUSSION THREAD

**Discussion by blaisep** [pypyr/pypyr#322] [Ideas]
I've been away for a while and I want to sketch up a few commands to handle a small Django app.
I can't remember where the examples project is so I go to the docs landing page and I notice that we don't consider someone who is NOT starting from zero and wants to skip right to the examples.

**Comment by yaythomas** [comment]
Hi @blaisep, great to have you back!

by docs landing page, do you mean: https://pypyr.io/docs/
and by example, are we talking about https://github.com/pypyr/pypyr-example?

**Reply by blaisep** [reply]
Yes, precisely. I suppose it could be something as simple as:
    _Been here before? Pypyr refresher shortcuts here..._
and a link to the popular resources.

**Reply by yaythomas** [reply]
check here https://pypyr.io/docs/#examples
